monkey/common/cloud/azure/azure_instance.py

```diff
@@ -41,7 +42,7 @@ def __init__(self):
             # If not on cloud, the metadata URL is non-routable and the connection will fail.
             # If on AWS, should get 404 since the metadata service URL is different, so bool(response) will be false.
             if response:
-                logger.debug("On Azure. Trying to parse metadata.")
+                logger.debug("Trying to parse metadata.")
```

Comment:
Why remove "On Azure"?

Reply:
The response which leads to this issue is not necessarily because it's on Azure and there's something wrong with the metadata. There may be no Azure instance at all.

Reply:
Still, should be more descriptive. ATM there's no indication what kind of metadata it is.
```suggestion
                logger.debug("Trying to parse Azure metadata.")
```